module-2/backend/week_one.md

```diff
@@ -5,30 +5,99 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
-2. What is Sinatra?
+    a. Get - used to view a page 
+    b. Post - Used to update a page 
+    c. Delete - deletes
+    d. Put - used to update a page
+    e. Patch - for partial updates
+2. What is Sinatra? 
+    Sinatra is a ruby framework to host programs on a server. In other words, its a tool used to allow a visual tool inside a 
+    browser. 
 4. What is MVC?
-5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
-6. What types of variables are accessible in our view templates without explicitly passing them?
+    a. Model
+    b. View 
+    c. Controller 
+    This is the file structure used in sinatra/ruby programs
+5. Why do we follow conventions when creating our actions/path names in our Sinatra routes? 
+      Because it allows for easy understanding across all programs. It is saved time that would otherwise be spend figuring  
+      out what extension goes to what page. 
+6. What types of variables are accessible in our view templates without explicitly passing them? 
+      yield

```

Comment:
`yield` isn't really a variable here. It's acting as a keyword. We were looking for *instance variables* (consistent with your answer to question 7).